src/Driver/Connection.php

```diff
@@ -57,7 +59,7 @@ public function exec(string $sql): int|string;
      * previous statement. DO NOT RELY ON THIS BEHAVIOR which is driver-dependent: always use getLastInsertId() right
      * after executing an INSERT statement.
      *
-     * @return int|string The last insert ID, as an integer or a numeric string.
+     * @return int|numeric-string The last insert ID, as an integer or a numeric string.
```

Comment:
I think this description is not accurate. What if the IDs are not represented by integers (like UUIDs)?

Reply:
Yes. Let's change the description of the return value or remove it entirely.

Reply:
This is addressed in #5909.